geoportailv3/models.py

```diff
@@ -17,6 +17,10 @@
 Base = sqlahelper.get_base()
 
 
+LayerWMTS.__acl__ = [DENY_ALL]
+WMTSDimension.__acl__ = [DENY_ALL]
```

Comment:
Because the changelog suppose that you steal use CGXP and those table aren’t used by the CGXP interface ...